sites.json

```diff
@@ -895,6 +915,17 @@
     },
 
     {
+        "meta": "popular",
```

Comment:
I can't speak for everyone, but I've personally never heard of ClassPass... Should this be marked as a popular website as such?